Formula/rizin.rb

```diff
@@ -0,0 +1,23 @@
+class Rizin < Formula
+  desc "UNIX-like reverse engineering framework and command-line toolset"
+  homepage "https://rizin.re"
+  url "https://github.com/rizinorg/rizin/releases/download/v0.3.1/rizin-src-v0.3.1.tar.xz"
+  sha256 "9ae4b7f9c47be63665fd0e59375b2b10f83c37156abb044ca4d61c1f0fc88f7e"
+  license "LGPL-3.0-only"
+  head "https://github.com/rizinorg/rizin.git", branch: "dev"
+
+  depends_on "meson" => :build
+  depends_on "ninja" => :build
+
+  def install
+    mkdir "build" do
+      system "meson", *std_meson_args, ".."
+      system "ninja"
+      system "ninja", "install"
+    end
+  end
+
+  test do
+    assert_match "rizin #{version}", shell_output("#{bin}/rizin -v")
```

Comment:
Can you propose a more substantial test than checking the version or calling for help? These tests rarely ever fail even for broken installations.

>We want tests that don’t require any user input and test the basic functionality of the application. For example `foo build-foo input.foo` is a good test and (despite their widespread use) `foo --version` and `foo --help` are bad tests. 

Ref. https://docs.brew.sh/Formula-Cookbook#add-a-test-to-the-formula

Reply:
Writing a more substantial test will likely requires an executable to disassemble. I’m not sure how to embed this in the formula, any suggestions or hints ?
Additionally I can’t think of a “short and straightforward” test that would be easy to maintain (I noticed tests in others formulas are usually one-liners)

Reply:
Can we `cp` something like `/usr/bin/true` and disassemble that? Or a copy of the `rizin` executable?

Reply:
I added more tests that doesn't require binaries, I hope it is acceptable 🙂